tasks/symlink-shared.yml

```diff
@@ -6,10 +6,19 @@
     path: "{{ ansistrano_release_path.stdout }}/{{ item }}"
   with_items: "{{ ansistrano_shared_paths }}"
 
+# Ensure symlink shared paths are existing
+- name: ANSISTRANO | Ensure shared paths directories are existing
+  file:
+    state: directory
+    src: "{{ ansistrano_shared_path.stdout }}/{{ item }}"
+    path: "{{ ansistrano_release_path.stdout }}/{{ item }}"
+  with_items: "{{ ansistrano_shared_paths }}"
+
 # Symlinks shared paths
 - name: ANSISTRANO | Create softlinks for shared paths
   file:
+    force: yes
     state: link
+    src: "{{ ansistrano_shared_path.stdout }}/{{ item }}"
```

Comment:
A little more absolute path imo is easier to track and probably less error prone than a regex.